examples/with-iron-session/lib/fetch/index.js

```diff
@@ -0,0 +1,25 @@
+import fetch from 'node-fetch'
```

Comment:
why are you using node-fetch for the swr fetcher since this is going to be ran on the client-side ?

Reply:
This fetcher is used both in api routes (nodejs) and browser code (pages via swr). Turns out node-fetch is just window.fetch in browsers (see https://github.com/node-fetch/node-fetch/blob/master/browser.js).

Also seems like "Next.js polyfills fetch for the browser" from @lfades

Reply:
To be clear this file should only be used for the server (as it's importing `node-fetch`), and I think that's the case for this case. If that's not the case then that import should be removed.